pages/index.js

```diff
@@ -1,7 +1,9 @@
+import CatList from "@/components/CatList/CatList";
+
 export default function HomePage() {
   return (
     <div>
```

Comment:
better use fragments here to not implement a div to the DOM tree

Reply:
thanks for your suggestion, I will change this :)

Reply:
changes are committed